css/awesome.css

```diff
@@ -37,15 +37,15 @@
 .awesome-board::before{
     position: absolute;
     content: "";
-    background-image: url("/picture/Group17&board.svg");
+    background-image: url("../picture/Group17&board.svg");
```

Comment:
Test comment for a file. Here you can reply to some notes from other developers.